security/keycloak-oauth2/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+# this will be overridden by an env var when deployed to OpenShift,
+# see SecurityKeycloakOpenShiftIT.configureKeycloakUrl
+quarkus.oauth2.introspection-url=http://localhost:8180/auth/realms/test-realm/protocol/openid-connect/token/introspect
+quarkus.oauth2.client-id=test-application-client
+quarkus.oauth2.client-secret=test-application-client-secret
+quarkus.oauth2.role-claim=roles
+
+quarkus.openshift.expose=true
+quarkus.s2i.base-jvm-image=registry.access.redhat.com/openjdk/openjdk-11-rhel7
```

Comment:
move on to registry.access.redhat.com/ubi8/openjdk-11

Reply:
Updated